blog/2023-05-05-new-pricing.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: new-pricing-2023-05
+title: 'New pricing is here. Including Pay as you go. Pay only for what you use.'
```

Comment:
How about: Introducing New Pricing Options: Pay as You Go, Only for What You Need!